### ISSUE TITLE
Template listing layout is distorted

### ISSUE THREAD

**Issue by imarklee** [joomla-projects/j4adminui#82]
### Steps to reproduce the issue
Head over to Extensions > Templates and click on the `Administrator` tab

### Expected result
Well arranged listing of templates

### Actual result
Distorted layout

<img width="1037" alt="template_listing_layout" src="https://user-images.githubusercontent.com/892204/69365404-3b3f7700-0ccf-11ea-946f-705e88b3c1fd.png">

**Comment by kawshar** [comment]
Because of the thumbnail resolution. Can be used as a background image. What do you think?

**Comment by imarklee** [comment]
Maybe template developers should prepare a screen shot with a specific aspect ratio or a min width / height so that the screenshots will look good on the template listings?

**Comment by kawshar** [comment]
Yes, that's why I keep that as it is. We need to set a standard resolution for the thumbnail.
